app/components/file/browse/display/file-list-display.html

```diff
@@ -1,37 +1,32 @@
-<bex-quick-list *ngIf="quickList">
-    <!--<bex-quick-list-item *ngFor="let node of nodes" [routerLink]="['/pools', poolId, 'nodes', node.id]" routerLinkActive="selected">-->
-    <!--<bex-quick-list-item *ngFor="let node of nodes" [routerLink]="['/pools', poolId, 'nodes', node.id]" routerLinkActive="selected">
-        <h4 bex-quick-list-item-title>{{node.id}}</h4>
-        <p bex-quick-list-item-field>{{node.state}}</p>
-    </bex-quick-list-item>-->
-    <h1>TODO: Implement quicklist</h1>
-</bex-quick-list>
+<bex-focus-section>
+    <bex-quick-list *ngIf="quickList">
+        <bex-quick-list-item *ngFor="let file of files| fileDirectoryFilter" [routerLink]="['/files', file.url]"
```

Comment:
Yup, good idea! I'll add that in later though. I'm wondering what to do w/ file types that we don't know about??? Or do we try to support icons for everything? For example .csv file? A .mov or .avi file? It might get a bit unwieldy... That said, there might be a library for that already... :)